acquia_cms.profile

```diff
@@ -88,16 +90,54 @@ function acquia_cms_install_tasks(): array {
  * Send heartbeat event after site installation.
  */
 function acquia_cms_send_heartbeat_event() {
+  $cohesion_configured = $config->get('api_key') && $config->get('organization_key');
+  // Get the install & rebuild time from state API.
+  $install_start_time = \Drupal::state()->get('install_start_time');
+  $install_end_time = \Drupal::state()->get('install_end_time');
+  $rebuild_start_time = \Drupal::state()->get('rebuild_start_time');
```

Comment:
@panshulK It doesn't seems to be working correct, when site is installed using drush command. Because cohesion rebuild happen using `drush post:command` hooks.

Reply:
Can we discus more on this? I can see that the function used to set the start time and finished time is being called by both drush (cli) and non cli executions.